python/tests/test_query.py

```diff
@@ -48,30 +51,66 @@ def table(tmp_path) -> MockTable:
 
 
 def test_query_builder(table):
-    df = LanceQueryBuilder(table, [0, 0]).limit(1).select(["id"]).to_df()
+    df = LanceQueryBuilder(table, [0, 0], "vector").limit(1).select(["id"]).to_df()
```

Comment:
shall we make `vector_column_name` to be a kwargs with default value, so that we don't break API like this.

Reply:
agreed - added default